apis/core/lib/domain/cube-projects/Project.ts

```diff
@@ -16,6 +16,7 @@ interface ApiProject {
   initializeJobCollection(store: ResourceStore): void
   incrementPublishedRevision(): void
   updateMaintainer(organization: Term | undefined): { before: Term; after: Term }
+  updateIsHiddenCube(hiddenCube: Term | undefined): void
```

Comment:
Would rather see this simplified. 

```suggestion
  updateIsHiddenCube(hiddenCube: boolean): void
```

But why a setter in the first place. You can already just set `isHiddenCube` directly